erpnext/accounts/doctype/pos_session_invoices/pos_session_invoices.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
Yes, the idea behind the child table was to have some sort of control log. 
For compliance reasons, we must ensure that if an invoice is deleted by an user, we have a trace of that deletion. Therefore I thought of having a link in the invoice and a log of all registered invoices in the POS Session. 
It was only a first step, since this log must also contain a SHA-2 key to guarantee it's inalterability.

I will delete the child table in this table for now and will make another PR for this log issue. Please let me know if you have any better idea of doing it.